tools/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+PROJECT(vrptw_drawer)
```

Comment:
let's call it `vrp_drawer` since there's `vrp_solver`. do not stick to a specific version of VRP, please